Modules/Core/Common/include/itkSingleton.h

```diff
@@ -64,20 +64,29 @@ class ITKCommon_EXPORT SingletonIndex
   }
 
 
-  // Returns true.
-  //
   // It is assumed that the global will remain valid until the start
   // of globals being destroyed.
   template <typename T>
-  bool
+  void
+  SetGlobalInstance(const char * globalName, T * global, std::function<void()> deleteFunc)
+  {
+    this->SetGlobalInstancePrivate(globalName, global, deleteFunc);
+  }
+
+#ifndef ITK_LEGACY_REMOVE
```

Comment:
```suggestion
#ifndef ITK_FUTURE_LEGACY_REMOVE
```

Reply:
@dzenanz  Honestly my question was just, do you want to change your review status from "Approved" to "Request changes"? Or do you think the current PR is still OK anyway?

Reply:
I encourage contributions, so I try not to make my nitpicks a barrier for merging. If people want to apply them, even better.

But I definitely misunderstood you here 😕 If a red symbol is what it takes to get the change, I will do it 😄